modules/vmseries/main.tf

```diff
@@ -106,13 +106,30 @@ resource "azurerm_virtual_machine" "this" {
   }
 }
 
-resource "azurerm_application_insights" "this" {
-  count = var.metrics_retention_in_days != 0 ? 1 : 0
+resource "azurerm_log_analytics_workspace" "this" {
+  count = try(var.app_insights_settings.create, false) && try(var.app_insights_settings.workspace_mode, true) ? 1 : 0
```

Comment:
Is `try(var.app_insights_settings.workspace_mode, true)` should be `try(var.app_insights_settings.workspace_mode, false)` ?

Reply:
The old (standard) app-insights will be depricated on Feb 29th 2024. So the module is using the app-insights with workspaces as the default set. This is the reason of having `True` here.